custom/icds_reports/reports/prevalence_of_undernutrition.py

```diff
@@ -33,7 +33,7 @@ def get_data_for(filters):
         if not show_test:
             queryset = apply_exclude(domain, queryset)
         if 'age_tranche' not in config:
-            queryset = queryset.exclude(age_tranche=72)
+            queryset = queryset.exclude(age_tranche__lt=72)
```

Comment:
same as above. this will exclude all children less than 72 month age.

Reply:
Done with the changes and updated the repository. Please have a look.